packages/bridge-ui-v2/scripts/exportJsonToEnv.js

```diff
@@ -31,8 +31,12 @@ jsonFiles.forEach((jsonFile) => {
     if (filename !== 'configuredChains') {
       base64Content = base64Content.replace(/\s+/g, '');
     }
-
-    const envKey = filename.replace(/([a-z0-9])([A-Z])/g, '$1_$2').toUpperCase();
+    
+    let envKey = filename.replace(/([a-z0-9])([A-Z])/g, '$1_$2').toUpperCase();
+    
+    if (envKey === 'CONFIGURED_CUSTOM_TOKENS') {
```

Comment:
What is this for?

Reply:
Thanks for checking out the pull request. That code change I made is for fixing an issue with the environment variable name that's generated by the `exportJsonToEnv.js` script. The script was creating a variable named `CONFIGURED_CUSTOM_TOKENS` from the `configuredCustomTokens.json` files name, but what we actually need according to our project setup (as shown in `.env.sample` and `generateCustomTokenConfig.ts`) is `CONFIGURED_CUSTOM_TOKEN`.

This mismatch was causing a problem where `CONFIGURED_CUSTOM_TOKEN` wasn't getting set right when I ran `pnpm export:config`, and it was messing up the `pnpm dev` command. So, I chnaged the script a bit to make sure it spits out the right variable name. I thought this was the simplest way to fix it without having to rename other things in the project.

Hope this clears things up, but if you need more info or have a different idea on how to handle it, just let me know.

Reply:
I chose this approach because it seemed the neatest. The alternatives would have involved either changing the variable name in `.env.example` and `generateCustomTokenConfig.ts` to `CONFIGURED_CUSTOM_TOKENS`, or renaming the file from `configuredCustomTokens.example` to `configuredCustomToken.example` and adjusting its path in `exportJsonToEnv.js`. This way, we maintain consistency without altering multiple components.

Reply:
It is already renamed to plural `configuredCustomTokens` in a different branch and will be merged into main soon. Please remove the related changes.

Reply:
Thank you for the update. I've reverted the changes related to configuredCustomTokens.